### DISCUSSION TITLE
Windows installation

### DISCUSSION THREAD

**Discussion by Salvodif** [rafaelmardojai/firefox-gnome-theme#881] [Q&A] [answered]
Hi, I installed the theme on Windows and I was trying to enable some
options in the "Enabling optional features" session but I can't find them.

By chance I can't find them because they are only added via the configuration file?
I did the installation by hand simply by copying the css files.
Thank you

**Comment by rafaelmardojai** [comment, marked as answer]
As the RADME explain (https://github.com/rafaelmardojai/firefox-gnome-theme?tab=readme-ov-file#enabling-optional-features) you need to create those settings:

1. Go to the `about:config` page
2. Type the key of the feature you want to enable
3. Set it as a `boolean` and click on the add button
